docs/development.rst

```diff
@@ -1108,6 +1108,13 @@ to him until directed).
          --retainIntermediates
 
 
+.. note::
+
+    The ``GeneticMap`` named ``"SalomeAveraged_TAIR10"`` for ``"AraTha"``
+    was generated by aligning the TAIR7 and TAIR10 with ``"minimap2"``,
+    and lifting the recombination rates on TAIR7 to TAIR10 with
+    ``"paftools.js liftover"``.
+
```

Comment:
Is this the level of detail you had in mind @petrelharp? Happy to add more, including commands, etc.

Reply:
Sounds good to me!

Reply:
The script in the issue is wonderful, though.